src/dsp_tools/commands/xmlupload/models/intermediary/values.py

```diff
@@ -70,6 +70,9 @@ class IntermediaryInterval(IntermediaryValue):
 class IntermediaryLink(IntermediaryValue):
     value: str
 
+    def __post_init__(self) -> None:
+        self.prop_iri = f"{self.prop_iri}Value"
```

Comment:
This seems a bit odd. I looked at the code and tried to understand why this is necessary, but I didn't manage.

Reply:
All links between two resources (eg. `LinkValue`) generate two link triples.

User-facing triple:
`:A :hasLinkTo :B`

1. `hasLinkTo` -> property that connects resource `:A` to resource `:B`
2.  `hasLinkToValue` -> property that connects resource `:A` to the `LinkValue` where all the metadata (permissions etc.) are stored.

Because we send metadata along, we do not send the actual `hasLinkTo` but the `hasLinkToValue`
I looked at the possible places where to do this best and because I want to save the correct data in the intermediary classes this seemed the easiest. But I am open to change this.